dash/webapp/pages/toggles.js

```diff
@@ -36,9 +36,15 @@ const featureToggles = [{
     'This will show unfilter those results, and allow for filtering.'
 }, {
   id: 'showWorkRedesign',
-  title: 'Show the new work page',
+  title: 'Show the work page prototype',
   description:
-    'Uses the new work page design.'
+    `Uses the work page prototype. Note, being in this group will take precedence ` +
+    `over the 'Show the single column work page' group.`
+}, {
+  id: 'showSingleColumnWork',
+  title: 'Show the single column work page',
```

Comment:
Could be more descriptive, maybe something like "Show work metadata in a single column"?